Packs/VirusTotal/Integrations/VirusTotal-V3-Premium-API/VirusTotal-V3-Premium-API.yml

```diff
@@ -0,0 +1,531 @@
+        file entries).
+      type: String
+  - arguments:
+    - default: true
+      description: A commma separated list of hashes (SHA-256, SHA-1, or MD5) for
+        the files included in the ZIP.
+      isArray: true
+      name: files
+      required: true
+      secret: false
+    - default: false
+      description: A password to protect the zip.
+      isArray: true
+      name: password
+      required: false
+      secret: true
+    deprecated: false
+    description: Creates a password-protected ZIP file containing files from VirusTotal.
+    execution: false
+    name: vt-premium-zip-create
+  - arguments:
+    - default: true
+      description: A zip ID. Can be retrieved from the output of vt-premium-zip-create.
+      isArray: false
+      name: zip_id
+      required: true
+      secret: false
+    deprecated: false
+    description: Retrieve information about a ZIP file.
+    execution: false
+    name: vt-premium-zip-get
+  - arguments:
+    - default: true
+      description: A zip ID. Can be retrieved from the output of vt-premium-zip-create.
+      isArray: false
+      name: zip_id
+      required: true
+      secret: false
+    deprecated: false
+    description: Download a ZIP file.
+    execution: false
+    name: vt-premium-zip-download
+  - arguments:
+    - default: true
+      description: Sandbox report ID. Can be aquired from vt-file-sandbox-report.
+      isArray: false
+      name: report_id
+      required: true
+      secret: false
+    deprecated: false
+    description: Extracted PCAP from a sandbox analysis.
+    execution: false
+    name: vt-premium-file-sandbox-pcap
+  - arguments:
+    - default: false
+      description: Search query.
+      isArray: false
+      name: query
+      required: true
+      secret: false
+    - default: false
+      description: Maximum number of results.
+      defaultValue: '10'
+      isArray: false
+      name: limit
+      required: false
+      secret: false
+    - default: false
+      description: The order value can be sorted depends on the query type. See documentation.
+      isArray: false
+      name: order
+      required: false
+      secret: false
+    - default: false
+      description: Continuation cursor.
+      isArray: false
+      name: cursor
+      required: false
+      secret: false
+    - default: false
+      description: Whether to return full object information or just object descriptors.
+      isArray: false
+      name: descriptors_only
+      defaultValue: 'false'
+      auto: PREDEFINED
+      predefined:
+      - "true"
+      - "false"
+      required: false
+      secret: false
+    deprecated: false
+    description: Search for files.
+    execution: false
+    name: vt-premium-intelligence-search
+  - arguments:
+    - default: true
+      description: Ruleset identifier.
+      isArray: false
+      name: id
+      required: true
+      secret: false
+    deprecated: false
+    description: Retrieve VT Hunting livehunt rulesets.
```

Comment:
```suggestion
    description: Retrieve VT Hunting livehunt ruleset.
```

Reply:
It's rulests, not ruleset. And it's a copy-paste from vt description